app_dart/bin/server.dart

```diff
@@ -55,7 +55,60 @@ Future<void> main() async {
       port: await config.forwardPort,
     );
 
+    /// Send HTTP 404 Response to Client
+    Future<void> sendNotFound(HttpResponse response) async {
+      response.statusCode = HttpStatus.notFound;
+      await response.close();
+    }
+
+    /// Send HTTP 500 Response to Client
+    Future<void> sendInternalError(HttpResponse response) async {
+      response.statusCode = HttpStatus.internalServerError;
+      await response.close();
+    }
+
+    /// Send static assets for the Flutter application to the client.
+    /// 
+    /// If the asset cannot be found, send a 404 Not Found response.
+    /// If there is an error sending the asset, send a 500 Internal Server Error
+    /// response.
+    Future<void> sendFlutterApplicationAssets(HttpRequest request) async {
```

Comment:
Rather than serving these directly in `bin/server.dart`, I'd rather make a `StaticFileHandler` class in https://github.com/flutter/cocoon/tree/master/app_dart/lib/src/request_handling and register a new `regexHandlers` map to live alongside the `handlers` map.

Reply:
That makes more sense. I started work here https://github.com/flutter/cocoon/blob/5b65714384081a3a4a03023e02f5207e7b45eff6/app_dart/lib/src/request_handling/static_file_handler.dart but do you think it is necessary for this PR to do the RegExp handling? Right now we just want to serve out of `/v2/*`.

Reply:
Yeah, it's fine to just wire up the static file handler straight to `/v2/*`